data/json/items/ammo.json

```diff
@@ -946,6 +946,7 @@
         "category" : "chems",
         "price" : 3500,
         "name" : "sprayable fungicide",
+        "name_plural" : "sprayable fungicide",
```

Comment:
Yes, fix for fungicide is useless because it's never shown in game, but it's shown as a string in game to translate in translator's Transifex interface. So after merging this PR, the incorrect string will never been seen again both by players and translators.